src/widgets/dialogs/UserInfoPopup.hpp

```diff
@@ -12,6 +15,10 @@ class QCheckBox;
 
 namespace chatterino {
 
+inline static const QString SEVENTV_USER_API =
+    "https://api.7tv.app/v2/users/%1";
+inline static const QString SEVENTV_CDR_PP = "https://cdn.7tv.app/pp/%1/%2";
```

Comment:
Not important at this time, and not likely to break, but adding as a note that this should use a url provided by the API. (But since v2 does not provide a URL in user responses, this is fine and can be amended when c7 is upgraded to the v3 api)